scripts/copy-files.js

```diff
@@ -0,0 +1,73 @@
+import path from 'path';
+import fse from 'fs-extra';
+
+const files = [
+  'README.md',
+  'CHANGELOG.md',
+  'LICENSE',
+];
+
+Promise.all(
+  files.map((file) => fse.copy(file, resolveBuildPath(file)))
+)
+.then(() => {
+  console.log('Copied files.'); // eslint-disable-line no-console
```

Comment:
Could perhaps include the content of the `files` const in this message, and 'to ./build', to be more specific about what was copied where.